recipes/botan/all/conanfile.py

```diff
@@ -0,0 +1,305 @@
+        'system_cert_bundle': "ANY"
+    }
+    default_options = {'amalgamation': True,
+                       'with_bzip2': False,
+                       'with_openssl': False,
+                       'shared': True,
+                       'fPIC': True,
+                       'single_amalgamation': False,
+                       'with_sqlite3': False,
+                       'with_zlib': False,
+                       'with_boost': False,
+                       'enable_modules': None,
+                       'system_cert_bundle': None}
+
+    def configure(self):
+        self._validate_compiler_settings()
+
+        if self.options.single_amalgamation:
+            self.options.amalgamation = True
+
+        if self.options.with_boost:
+            self.options["boost"].add("shared=False")
+            self.options["boost"].add("magic_autolink=False")
+            self.options["boost"].add("without_coroutine=False")
+            self.options["boost"].add("without_system=False")
+
+    def requirements(self):
+        if self.options.with_bzip2:
+            self.requires("bzip2/1.0.6")
+        if self.options.with_openssl:
+            self.requires("openssl/1.0.2t")
+        if self.options.with_zlib:
+            self.requires("zlib/1.2.11")
+        if self.options.with_sqlite3:
+            self.requires("sqlite3/3.30.1")
+        if self.options.with_boost:
+            self.requires("boost/1.71.0")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "botan-" + self.version
+        os.rename(extracted_dir, "sources")
+
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        with tools.chdir('sources'):
+            self.run(self._configure_cmd)
+            self.run(self._make_cmd)
+
+    def package(self):
+        self.copy(pattern="license.txt", dst="licenses", src="sources")
+        with tools.chdir("sources"):
+            self.run(self._make_install_cmd)
+
+    def package_info(self):
+        if self.settings.compiler == 'Visual Studio':
+            self.cpp_info.libs.append('botan')
+        else:
+            self.cpp_info.libs.extend(['botan-2'])
+            if self.settings.os != 'Windows':
+                self.cpp_info.system_libs.append('dl')
+            if self.settings.os == 'Linux':
+                self.cpp_info.system_libs.append('rt')
+            if self.settings.os == 'Macos':
+                self.cpp_info.frameworks = ['Security', 'CoreFoundation']
+            if not self.options.shared:
+                self.cpp_info.system_libs.append('pthread')
+        if self.settings.os == "Windows":
+            self.cpp_info.system_libs.extend(["ws2_32", "Crypt32"])
+
+        self.cpp_info.libdirs = ['lib']
+        self.cpp_info.bindirs = ['bin']
```

Comment:
It is ok, but those dirs are the default ones, so there is no need to specify them here

Reply:
Ah nice, I'll remove this :)